scancodeio/settings.py

```diff
@@ -414,3 +414,7 @@
 MATCHCODEIO_USER = env.str("MATCHCODEIO_USER", default="")
 MATCHCODEIO_PASSWORD = env.str("MATCHCODEIO_PASSWORD", default="")
 MATCHCODEIO_API_KEY = env.str("MATCHCODEIO_API_KEY", default="")
+
+# OpenSSF ScoreCard Integration
+
+SCORECARD_URL = env.str("SCORECARD_URL", default="")
```

Comment:
I don't think this is likely to change that much? We use URLs in settings when there can be many instances for the same. Maybe we should have a default?

Reply:
yup removed it